### PR TITLE
Propagate builtin from Primitive to Cextcall

### PR DESCRIPTION
**Pull request by gretay-js**
on top of #11 and #10, only the top commit is new.

### CODE REVIEW

**Comment by gretay-js** [comment]
Splitting up effects and coeffects into a separate PR.

File: backend/cmm_helpers.mli [hunk 1]
```diff
@@ -570,6 +570,12 @@ val send :
   Lambda.meth_kind -> expression -> expression -> expression list ->
   Debuginfo.t -> expression
 
+(** [transl_cextcall prim args dbg type_of_result] returns Cextcall operation
```
Comment (mshinwell) [inline backend/cmm_helpers.mli:573]: transl_cextcall -> cextcall

File: backend/cmm_helpers.mli [hunk 2]
```diff
@@ -570,6 +570,12 @@ val send :
...
+    that corresponds to [prim]. If [prim] is a builtin supported on the target,
```
Comment (mshinwell) [inline backend/cmm_helpers.mli:574]: maybe "builtin" -> "C builtin" to clarify

**Review by mshinwell — CHANGES_REQUESTED** [review]

**Comment by gretay-js** [comment]
I fixed the comment, thanks!  
Squashed the commits.

**Review by mshinwell — APPROVED** [review]

**Pull request merged**